tests/dragonfly/replication_test.py

```diff
@@ -2999,7 +2998,7 @@ async def test_replica_snapshot_with_big_values_while_seeding(df_factory: DflyIn
     c_replica = replica.client()
 
     # 50% big values
-    seeder_config = dict(key_target=20_000, huge_value_target=10000)
+    seeder_config = dict(key_target=8_000, huge_value_target=4_000)
```

Comment:
Ok for this change, though I suspect it will not be stable for long because you check the side_saved. I suggest to follow up with a PR that will add a flag to yield on x buckets granularity in snapshot as we have in migration